app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
Continuing conversation from https://github.com/ManageIQ/manageiq-automation_engine/pull/35#discussion_r120493827.

The caller must know the VM so I feel like these configure methods belong on the VM and not the provider.  I know this is intended for imported VMs but would assume that it would work on any VM so the naming again seems off.

What I would like to understand is if this method can be moved to the appropriate VM model (called without `vm_id`) and renamed to something like `configure_networks`?  Then the method it calls would also be moved and renamed to something better.